ui/app/models/ldap/library.js

```diff
@@ -81,4 +84,23 @@ export default class LdapLibraryModel extends Model {
   get canList() {
     return this.libraryPath.get('canList') !== false;
   }
+  get canReadStatus() {
+    return this.statusPath.get('canRead') !== false;
+  }
+  get canCheckOut() {
```

Comment:
Just confirming this is not update here? Not every endpoint is the same, and I know for example with KV POST on metadata it requires update. Vault policy doc [here](https://developer.hashicorp.com/vault/docs/concepts/policies#capabilities).

Reply:
That's a good question. I initially thought `create` should map to `POST` but there are plenty of instances where we update something through `POST`. Wouldn't it be dependent on the policy either way though? If I changed it `update` and the user has a policy with `create` but not `update` for the endpoint then they wouldn't have access.

To me it seems like `create` is appropriate because we are not updating anything on the library itself but rather creating credentials for an account to be used while it's checked out. That being said, since it is a bit ambiguous here maybe I should check for either `create` or `update` capabilities?

Reply:
The line in the doc link i sent is the issue:
> [create](https://developer.hashicorp.com/vault/docs/concepts/policies#create) (POST/PUT) - Allows creating data at the given path. Very few parts of Vault distinguish between create and update, so most operations require **both** create and update capabilities. Parts of Vault that provide such a distinction are noted in documentation.

Reply:
Yep you are totally right! Having only create fails but only update works. Thanks for pointing this out I haven't come across a situation quite like this before!

I have a follow up task to acceptance test the various workflows so I think it's worthwhile to have coverage for permissions. In the past I have stubbed the capabilities checks but that isn't fully reliable in this case where the backend would fail the request for a policy with create only.

Reply:
Interestingly though, the request was a success when I added had a policy with `update` but I didn't actually change the capabilities check in the model 😕